### PR TITLE
Add a row to the Admin dashboard summary of supervisors for volunteers not assigned to supervisors

### PR DESCRIPTION
**Pull request by ErinClaudio**
### What github issue is this PR for, if any?
WIP #1400 

### What changed, and why?
Nothing of note

### How will this affect user permissions?
- Volunteer permissions:
- Supervisor permissions:
- Admin permissions:

### How is this tested? (please write tests!) 💖💪
Nothing real just a placeholder test

### Screenshots please :)
Screen Shot 2021-04-07 at 8 46 23 PM

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### CODE REVIEW

**Comment by ErinClaudio** [comment]
@compwron, this is interesting. I will ask you about it this evening.

File: app/controllers/supervisors_controller.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@ class SupervisorsController < ApplicationController
   def index
     authorize Supervisor
     @supervisors = policy_scope(current_organization.supervisors)
+    available_volunteers
```
Comment (compwron) [inline app/controllers/supervisors_controller.rb:12]: before_action

File: app/views/supervisors/index.html.erb [hunk 2]
```diff
@@ -56,3 +56,24 @@
...
+      <tr>
+        <th>Volunteers not assigned to supervisors</th>
+      </tr>
+    </thead>
+      <tbody>
+        <td>
+          <% if @available_volunteers.count <= 0 %>
+            <%= "Currently no volunteers are unassigned"%>
+          <% else %>
+            <%  @available_volunteers.active.each do |volunteer| %><br>
```
Comment (compwron) [inline app/views/supervisors/index.html.erb:73]: Unassigned volunteers:

File: spec/system/supervisors/index_spec.rb [hunk 3]
```diff
@@ -86,5 +86,10 @@ def verify_numeric_sort(column)
     it "by no-contact count", js: true do
       verify_numeric_sort("No Contact (14 days)")
     end
+
+    it "will not show a list of volunteers not assigned to supervisors if the
```
Comment (compwron) [inline spec/system/supervisors/index_spec.rb:90]: add positive test

**Comment by compwron** [comment]
For failing tests in https://github.com/rubyforgood/casa/blob/main/spec/views/volunteers/index.html.erb_spec.rb
Add an assign
like existing `assign :volunteers, [volunteer]`

File: app/views/supervisors/index.html.erb (hunk 2 above)
Comment (compwron) [reply app/views/supervisors/index.html.erb:73]: from office hours 

```

  context "with unassigned volunteers" do
    let!(:unassigned_volunteer) { create(:volunteer, casa_org: organization, display_name: "Tony Ruiz") }

    before do
      sign_in user
      visit supervisors_path
    end

    it "will show a list of unassigned volunteers" do
      expect(page).to have_text("Unassigned volunteers:")
      expect(page).to have_text("Tony Ruiz") # test link too
      expect(page).not_to have_text("Currently no volunteers are unassigned")
    end
  end

  context "without unassigned volunteers" do
    before do
      sign_in user
      visit supervisors_path
    end
    it "will not show a list of volunteers not assigned to supervisors" do
      expect(page).to have_text("Currently no volunteers are unassigned")
    end
  end
end
```

Comment (compwron) [reply app/views/supervisors/index.html.erb:73]: I assigned this to you :) https://github.com/rubyforgood/casa/issues/1990

Comment (ErinClaudio) [reply app/views/supervisors/index.html.erb:73]: Thank you @compwron

**Comment by ErinClaudio** [comment]
@compwron 

1. I will have a question about merge conflicts. I attempted to resolve this merge conflict through a git push of code. I do not think this is correct. I will need to do some research on this first.
2. The number of checks for CI seems to be much lower ( from 8ish to 2). Curious as to why that is?

**Comment by compwron** [comment]
> @compwron
> 
> 1. I will have a question about merge conflicts. I attempted to resolve this merge conflict through a git push of code. I do not think this is correct. I will need to do some research on this first.
> 2. The number of checks for CI seems to be much lower ( from 8ish to 2). Curious as to why that is?

1. 
https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line
https://www.simplilearn.com/tutorials/git-tutorial/merge-conflicts-in-git

2. that's strange. Where are you seeing that? Link or screenshot? Possibly the checks won't run until the merge conflict is resolved.

**Comment by ErinClaudio** [comment]
@compwron Thank you for the docs

**Review by compwron — APPROVED** [review]
let's merge! There's a test failure, we'll fix it

**Pull request merged**